Library/Homebrew/cmd/style.rb

```diff
@@ -111,12 +111,18 @@ def check_style_impl(files, output_type, options = {})
       args << "--only" << cops_to_include.join(",")
     end
 
+    if files.nil? || ARGV.include?("--rspec")
+      config_file_base = ARGV.include?("--rspec") ? ".rubocop-rspec.yml" : ".rubocop.yml"
```

Comment:
Think you could just `if`/`else` this without variables and use `HOMEBREW_LIBRARY_PATH` in both?

Reply:
Amended to fix.